src/managers/cog_system.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+class _CogSystemManager:
+    def __init__(self, client):
+        self.client = client
+
+    async def reg_debug_cogs(self):
+        # Remote code execution module for debugging purposes
+        await self.client.load_extension("jishaku")
+
+    async def reg_cogs(self):
+        # TODO: Ensure this is only loaded in debug mode, not in live
+        await self.reg_debug_cogs()
+
+        for filename in os.listdir("./src/cogs"):
+            if filename.endswith(".py"):
+                await self.client.load_extension(f"src.cogs.{filename[:-3]}")
+
+
+# Typedef for clarity, make sure to import CogSystem
+CogSystem = _CogSystemManager
```

Comment:
I am aware this looks ugly, however namespaces like in C++:
```cpp
using CogSystem = detail::CogSystemManager;

namespace detail
{
    class CogSystemManager
    {
        ...
    };
}
```

Is not possible, I could of create a module and used that to namespace it, but that is more obscure. Opted for this due to its simplicity. Thoughts?

The `_` indicates this shouldn't be referenced by the user, on top of that they should import the _'typedef'_ `CogSystem`. As shown in `main.py`.